nntrainer/layers/conv2d_layer.cpp

```diff
@@ -484,7 +490,11 @@ void Conv2DLayer::calcGradient(RunLayerContext &context) {
   }
 
   delK.reshape(filter_dim);
-  derivative.sum({0, 2, 3}, delBias);
+  if (auto &disable_bias = std::get<props::DisableBias>(*layer_impl_props);
+      disable_bias.empty() || disable_bias.get() == false) {
```

Comment:
nit picking) constructor of props::DisableBias that it is not empty so IMO we wouldn't need to check disable_bias.empty() for all.